### ISSUE TITLE
Keys shifted in VIA

### ISSUE THREAD

**Issue by haoxiangliew** [the-via/releases#169]
In my attempt to port the Keyboardio Atreus from QMK to Via, it appears that a group of keys cannot be detected by VIA, but work fine in QMK.

Directory for reference:
[https://github.com/haoxiangliew/qmk_firmware/tree/master/keyboards/keyboardio/atreus/keymaps/via](url)

The following VIA .json for the Atreus, and it's matrix map defined in the "Design" tab, matches the header file `atreus.h` found in the QMK directory.
[https://github.com/haoxiangliew/qmk_firmware/blob/master/keyboards/keyboardio/atreus/keymaps/via/keyboard.json](url)
![image](https://user-images.githubusercontent.com/62730710/157590774-a7d2e004-19c8-437e-8ce5-688b9348cc67.png)
[https://github.com/haoxiangliew/qmk_firmware/blob/master/keyboards/keyboardio/atreus/atreus.h](url)
![image](https://user-images.githubusercontent.com/62730710/157590950-024c8c22-e771-4a8d-988c-7bde102a1e56.png)

The revisions for adding VIA support for QMK can be found in the following commit:
[https://github.com/haoxiangliew/qmk_firmware/commit/ddbd5740706a78efcbe181ec356a59ec268c9321](url)

1. Keymap added with KC_TRNS 4th layer
2. VIA .json added with correct VENDOR and PRODUCT ID
3. `rules.mk` with `VIA_ENABLE = yes` and `LTO_ENABLE = yes`

With these changes, the QMK portion of the keymap works fine, and all keys correspond with `keymap.c`
![image](https://user-images.githubusercontent.com/62730710/157591494-b4ac642e-95c8-4825-a09d-4da05654e0dd.png)

However, VIA's key tester with the matrix shown tells a different story.
![image](https://user-images.githubusercontent.com/62730710/157591616-bdc59f39-9fdb-40c1-8787-da3efbfce95b.png)

4 keys are incorrectly offset on Rows 0 and 1, as shown by VIA
![image](https://user-images.githubusercontent.com/62730710/157591682-45f8b1d4-3ee0-40fb-bc49-44862833e272.png)

Basically, according to the matrix map
```c
#define LAYOUT(                                                  \
  k00, k01, k02, k03, k04,           k05, k06, k07, k08, k09,    \
  k10, k11, k12, k13, k14,           k15, k16, k17, k18, k19,    \
  k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b,    \
  k30, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b     \
)
```
is now
```c
#define LAYOUT(                                                  \
  k00, k01, k02, k03, k04,           k07, k08, k09, KC_NO, KC_NO,    \
  k10, k11, k12, k13, k14,           k17, k18, k19, KC_NO, KC_NO,    \
  k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b,    \
  k30, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b     \
)
```
with the `O`, `P`, `L` and `;` keys useless.

**Comment by haoxiangliew** [comment]
Fixed with json matrix offset! Not an issue with Via.
